docs/csharp/programming-guide/concepts/async/index.md

```diff
@@ -26,6 +26,10 @@ Now, consider those same instructions written as C# statements:
 
 :::code language="csharp" source="snippets/index/AsyncBreakfast-starter/Program.cs" highlight="8-27":::
 
+:::image type="content" source="media/synchronous-breakfast.png" alt-text="synchronous breakfast":::
+
+The synchronously prepared breakfast, took roughly 30 minutes because the total is simply the sum of each individual task.
```

Comment:
nit: remove simply:

```suggestion
The synchronously prepared breakfast, took roughly 30 minutes because the total is the sum of each individual task.
```